Hints.py

```diff
@@ -750,9 +755,34 @@ def buildWorldGossipHints(spoiler, world, checkedLocations=None):
 
         if world.shopsanity != "off" and "Progressive Wallet" not in world.item_hints:
             world.item_hints.append("Progressive Wallet")
-        world.named_item_pool = list(world.item_hints)
-
+                
+
+    #Removes items from item_hints list if they are included in starting gear.
+    #This method ensures that the right number of copies are removed, e.g.
+    #if you start with one strength and hints call for two, you still get
+    #one hint for strength
+    for item in [*world.starting_items, *world.starting_equipment, *world.starting_songs]:
```

Comment:
I would tend to prefer itertools.chain for this.

Reply:
Did not know that was a thing, but that is exactly what I was looking for here. Thanks! :D  Made the switch